metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/importer/api/SraObjectsImporterThroughApiTest.java

```diff
@@ -198,17 +197,17 @@ public void importReferenceSequenceGcfAssemblyAccessionWithoutApiKey() throws Ex
     @Test
     @Category(EntrezApiKeyAccessCategory.class)
```

Comment:
i guess you can remove this category then and dlete the class

Reply:
You're right, removed in https://github.com/EBIvariation/metadata-ws/pull/72/commits/6f9790ee8eaf765728ee2eadf486f9f1ed036a3a